easybuild/easyblocks/c/clang.py

```diff
@@ -85,6 +85,8 @@ def extra_options():
             'skip_all_tests': [False, "Skip running of tests", CUSTOM],
             # The sanitizer tests often fail on HPC systems due to the 'weird' environment.
             'skip_sanitizer_tests': [True, "Do not run the sanitizer tests", CUSTOM],
+            'default_cuda_capability': [None, "Default CUDA capability specified for clang, e.g. '7.5'", CUSTOM],
+            'cuda_compute_capabilities': [[], "List of CUDA compute capabilities to build with", CUSTOM],
```

Comment:
This is already a general easyconfig parameter (since https://github.com/easybuilders/easybuild-framework/pull/3382), so adding `cuda_compute_capabilities` here too shouldn't be needed...

@nordmoen Do you have a `Clang` easyconfig file that requires the enhancement you implemented here?

Reply:
@boegel The default parameter is quite nice if a user has specified more than one compute compatibility and doesn't want to use the minimum.

I added the `cuda_compute_capabilities` based on [feedback from @Flamefire on Slack](https://easybuild.slack.com/archives/C34UA1HT7/p1605097924389300?thread_ts=1605082171.371900&cid=C34UA1HT7). In this way it resembles other similar `easyconfig` files like `TensorFlow`. I have used the built-in `--cuda-compute-capabilites` flag in EB and for me it should not be needed.

Reply:
There's two different things here:

1) The `cuda_compute_capabilities` easyconfig parameter, which is a general easyconfig parameter (so you don't need to define it in `extra_options` for it to be defined & picked up), see "`eb -a | grep cuda`".

2) The `--cuda-compute-capabilities` EasyBuild configuration option, which is what you're picking up via `build_option('cuda_compute_capabilities')` (I know it can be confusing...).

I was merely pointing out that add `cuda_compute_capabilities` as a knwon easyconfig parameter is not needed at all, since it's already a known general easyconfig parameter. There's no real downside though (except for adding to the confusion, maybe).